backend/app/api/endpoints/internal/chat_storage.py

```
@@ -257,33 +259,49 @@ def _build_user_message_content(
     attachment_text_parts: list[str] = []
     total_attachment_text_length = 0
 
-    for attachment in attachments:
+    for idx, attachment in enumerate(attachments, start=1):
```

Ruff (0.14.13)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)